_guides/scheduling-flow-using-cron.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Scheduling a flow with CRON
+layout: article
+section: Building integration flows
+category: component
+order: 4
+since: 20181218
+---
+
+Scheduling tasks are part of the Elastic Platform's functionality. In this article, we are going to cover the scheduling scenario using CRON. 
+
+## What is CRON 
+
+[CRON](https://en.wikipedia.org/wiki/Cron) is the UNIX-like job scheduler that executes commands or scripts at given time and date. 
+CRON is driven by a *cron table*, to run periodically on a given schedule. See the *crontab* line on the screenshot below:
+
+![Crontab](/assets/img/developer-guide/cron/cron.png "Crontab")
+
+> **Please Note**: By default, our System repeats the flow's execution every 3 minutes. However, you are free to customize the time range according to your needs.
+
+The line's syntax expects a CRON expression made of five values. For more information on how to generate CRON's expressions, please follow this [link](https://www.freeformatter.com/cron-expression-generator-quartz.html).
+
+## The Scheduling
+
+Please be aware, that changing the flow's schedule means modifying the flow. Here some of the requirements to follow: 
+
+*   For modifying a flow, you should have specific permissions.
+*   As soon as you modify the flow's schedule, you should restart it to let the System apply the changes.
+
+To proceed further with CRON scheduling follow the steps below:
+
+**First step**
+
+Create a [flow](#realtime-flows) or pick the existing one by navigating to the **Workspace > Flows > Choose the necessary Flow > Settings** tab. See the screenshot below.
+
+![CRON Navigation](/assets/img/developer-guide/cron/cron_navigation.png "CRON Navigation")
+
+**Second step**
```

Comment:
Now that you showed how to use the feature, we can concentrate on the last detail - the CRON expression itself. We should mention in the beginning that teaching the reader how to use CRON properly goes beyond the scope of this article and link the corresponding CRON docs. However we should give here 5-10 most realistic CRON expression and discuss them in details: explaining the expression itself and showing its execution plan on the right hand side.

Reply:
Did that.